evalml/pipelines/utils.py

```diff
@@ -134,7 +157,9 @@ def make_pipeline(X, y, estimator, problem_type, parameters=None, custom_hyperpa
     problem_type = handle_problem_types(problem_type)
     if estimator not in get_estimators(problem_type):
         raise ValueError(f"{estimator.name} is not a valid estimator for problem type")
-    preprocessing_components = _get_preprocessing_components(X, y, problem_type, estimator)
+    if not is_classification(problem_type):
```

Comment:
I think this should probably raise an exception when `problem_type` and `sampler` conflicts.